test/services/modules/ModuleDaoSpec.scala

```diff
@@ -41,14 +39,6 @@ object ModuleDaoSpec extends Specification {
   }
 
   def withDao[T](block: DbModuleDao => T) =
-    Databases.withDatabase(
-      driver = "com.mysql.jdbc.Driver",
-      url = "jdbc:mysql://localhost:3306/playunittest",
-      config = Map("user" -> "root"),
-    ) { db =>
-      Evolutions.withEvolutions(db) {
-        block(new DbModuleDao(db))
-      }
-    }
+    block(new DbModuleDao())
```

Comment:
at first, I thought that this was a blocking operation. 

Maybe we should just call if `func`